### ISSUE TITLE
 Distributor script execution additionalParameters feature causing failures

### ISSUE THREAD

**Issue by deric4** [aws/amazon-ssm-agent#296]
The following command used to work, but is now failing and produces an error since the release of https://github.com/aws/amazon-ssm-agent/releases/tag/2.3.1550.0 

```shell-session
$ aws ssm send-command --document-name AWS-ConfigureAWSPackage --instance-ids <instance id> --parameters action=Install,name=AmazonCloudWatchAgent

...

$ aws ssm list-command-invocations \                                                                                                                     
        --command-id <command-id> \
        --details
{
    "CommandInvocations": [
        {
            "CommandId": "<command id>",
            "InstanceId": "<instance id>",
            "InstanceName": "",
            "Comment": "",
            "DocumentName": "AWS-ConfigureAWSPackage",
            "DocumentVersion": "",
            "RequestedDateTime": "2020-07-30T19:55:09.462000-07:00",
            "Status": "Failed",
            "StatusDetails": "Failed",
            "StandardOutputUrl": "",
            "StandardErrorUrl": "",
            "CommandPlugins": [
                {
                    "Name": "configurePackage",
                    "Status": "Failed",
                    "StatusDetails": "Failed",
                    "ResponseCode": 1,
                    "ResponseStartDateTime": "2020-07-30T19:55:09.870000-07:00",
                    "ResponseFinishDateTime": "2020-07-30T19:55:09.873000-07:00",
                    "Output": "\n----------ERROR-------\ninvalid format in plugin properties map[action:Install additionalArguments:map[] installationType:Uninstall and reinstall name:AmazonCloudWatchAgent version:]; \nerror json: cannot unmarshal object into Go struct field ConfigurePackagePluginInput.additionalArguments of type string\n",
                    "StandardOutputUrl": "",
                    "StandardErrorUrl": "",
                    "OutputS3Region": "us-west-2",
                    "OutputS3BucketName": "",
                    "OutputS3KeyPrefix": ""
                }
            ],
            "ServiceRole": "",
            "NotificationConfig": {
                "NotificationArn": "",
                "NotificationEvents": [],
                "NotificationType": ""
            },
            "CloudWatchOutputConfig": {
                "CloudWatchLogGroupName": "",
                "CloudWatchOutputEnabled": false
            }
        }
    ]
}
```

Quoting and explicitly adding the `additionalArguments` to doc parameters seems to work:

```shell-session
$ aws ssm send-command --document-name AWS-ConfigureAWSPackage --instance-ids <instance ids> --parameters "action=Install,name=AmazonCloudWatchAgent,additionalArguments=\"{}\""
```

**Comment by VishnuKarthikRavindran** [comment]
Thanks @deric4 for reaching out. We are working on this and the fix will be out soon. We also appreciate your contribution towards this issue.

**Comment by sahanracker** [comment]
It seems like this affect to lot of audiences and our automation scripts failing due to this issue. Any ETA for this or workaround?

**Comment by ferkhat-aws** [comment]
This has been fixed in version 2.3.1613.0.
Closing.
